src/js/components/Select/SelectContainer.js

```diff
@@ -269,6 +269,7 @@ const SelectContainer = forwardRef(
           ) {
             setActiveIndex(nextActiveIndex);
             setFocusWithoutScroll(clearButton);
+            setFocusWithoutScroll(searchInput);
```

Comment:
if we change this then we are unable to ever clear options again.

https://github.com/user-attachments/assets/cd9dac4b-5950-4bf6-a1d2-eae18110fa28

for reference I was testing with the following story you can copy and paste in simple.js

```
import React, { useState } from 'react';

import { Box, Select } from 'grommet';

const defaultOptions = [];
for (let i = 1; i <= 10; i += 1) {
  defaultOptions.push(`option ${i}`);
}

export const Simple = () => {
  const [options, setOptions] = useState(defaultOptions);
  const [value, setValue] = useState('');
  return (
    // Uncomment <Grommet> lines when using outside of storybook
    // <Grommet theme={...}>
    <Box fill align="center" justify="start" pad="large">
      <Select
        size="medium"
        placeholder="Select single option"
        value={value}
        options={options}
        onChange={({ option }) => setValue(option)}
        onClose={() => setOptions(defaultOptions)}
        onSearch={(text) => {
          const escapedText = text.replace(/[-\\^$*+?.()|[\]{}]/g, '\\$&');
          const exp = new RegExp(escapedText, 'i');
          setOptions(defaultOptions.filter((o) => exp.test(o)));
        }}
        clear
      />
    </Box>
    // </Grommet>
  );
};

Simple.args = {
  full: true,
};

export default {
  title: 'Input/Select/Simple',
};

```

this code is just for testing we do not want to change the story but if this helps feel free to use locally